frontend/src/components/LandingPage/TeamCard/TeamCard.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {
+  Card, /*CardImg, CardText,*/ CardBody,
+  CardTitle, CardSubtitle, /*Button*/
+} from 'reactstrap';
+import Image from 'react-bootstrap/Image';
+import './TeamCard.css'
+
+const TeamCard = (props) => {
```

Comment:
Can you convert this into a functional component instead of a constant.

Reply:
Done